compiler/src/dotty/tools/dotc/core/CheckRealizable.scala

```diff
@@ -66,22 +66,53 @@ class CheckRealizable(implicit ctx: Context) {
    */
   private def isLateInitialized(sym: Symbol) = sym.is(LateInitialized, butNot = Module)
 
-  /** The realizability status of given type `tp`*/
+  /** The realizability status of given type `tp`. Types can only project
+    * members from realizable types, that is, types having non-null values and
+    * not depending on mutable state.
+    * Beware that subtypes of realizable types might not be realizable; hence
+    * realizability checking restricts overriding. */
   def realizability(tp: Type): Realizability = tp.dealias match {
```

Comment:
@Blaisorblade what is the difference/relation between a type being stable vs being realizable? Consider adding that information to the method documentation.